dynawo/sources/Models/CPP/ModelNetwork/DYNModelNetwork.cpp

```diff
@@ -1426,4 +1426,12 @@ ModelNetwork::setGequations() {
   }
 }
 
+void
+ModelNetwork::printInitValuesParameters(std::ofstream& fstream) {
+  for (vector<shared_ptr<NetworkComponent> >::const_iterator itComponent = components_.begin();
```

Comment:
I would prefer to avoid displaying n times the parameters that are common to all members of a type. 
For example load_isControllable should be displayed once, + as many <LOAD_ID>_isControllable that were defined.

Reply:
Each of these parameters can have a different value

Reply:
load_isControllable is unique for all loads, whereas id_isControllable is specific for a single load

Reply:
With the previous code, network common parameters are not printed. With new code, default behaviour ios that common parameters are printed but not the ones in the models. To discuss for another PR